src/web/components/Nav/Nav.tsx

```diff
@@ -166,8 +166,6 @@ export const Nav = ({ format, nav, subscribeUrl, edition }: Props) => {
 					rowStyles,
 					format.display === Display.Immersive && minHeight,
 				]}
-				role="navigation"
-				aria-label="Guardian sections"
```

Comment:
I'm deleting these because the child components on `nav` (the ones we land on when tabbing) already have their own labels so having these as well means both are spoken